torch/distributions/relaxed_categorical.py

```diff
@@ -30,8 +30,8 @@ class ExpRelaxedCategorical(Distribution):
     (Jang et al, 2017)
     """
     arg_constraints = {'probs': constraints.simplex,
-                       'logits': constraints.real}
-    support = constraints.real
+                       'logits': constraints.real_vector}
+    support = constraints.real_vector
```

Comment:
cc. @fritzo.

Reply:
LGTM. I'm not sure whether `real_vector` is the optimal constraint, but I believe it is the tightest valid constraint that we currently implement. I think that's fine since most users will use the wrapper class with the exact `simplex` constraint.